client.go

```diff
@@ -37,33 +44,72 @@ func NewClient(c Config) (*Client, error) {
 	}
 
 	return &Client{
-		client: goredis.NewClient(&goredis.Options{
-			Addr:     c.Addr,
+		client:  RedisClient{client:redis.NewClient(&redis.Options{
+			Addr: c.Addr,
 			Password: c.Password,
-			DB:       c.Database,
-		}),
-		ttl: c.TTL,
+			DB: c.Database,
+		})},
+		ttl:     0,
 	}, nil
 }
 
-//Set add session to redis
+// Set - add session to redis
```

Comment:
👇 We have an unofficial coding standard whereby the first argument for a client should always be `context`.